migration/idempotent/013-apply-config.sql

```diff
@@ -1,13 +1,8 @@
 
 DO $block$
 DECLARE
-    _old_search_path text;
     _rec record;
 BEGIN
-    -- set the search_path such that pg_get_function_identity_arguments
-    -- should return fully schema qualified arguments
-    EXECUTE 'SHOW search_path' INTO STRICT _old_search_path;
-    SET search_path TO pg_temp;
```

Comment:
why was this removed?

Reply:
I should have added the justification behind this to the readme. In the migration and idempotent wrappers we wrap the script with:

```SQL
    EXECUTE 'SHOW search_path' INTO STRICT _old_search_path;
    SET search_path TO pg_catalog;

-- Note: this weird indentation is important. We compare SQL across upgrade paths,
-- and the comparison is indentation-sensitive.
{{body}}

    EXECUTE format('SET search_path TO %s', _old_search_path);
```

Which sets the search_path for the whole block, in the same manner that this mechanism does.